docs/user-guide/installation.rst

```diff
@@ -16,9 +16,12 @@ in order to implement an API compatible with Puppet Forge. We don't like
 taking that namespace, but it was the only way to support the use of Puppet
 Labs' command line tool against a Pulp server.
 
-Consumers must have Puppet 2.7.14 to 3.2.4 installed, and we recommend getting packages
-directly from `Puppet Labs <http://puppetlabs.com>`_.  Pulp is not compatible with Puppet
-3.3.x.
+Consumers must have Puppet 2.7.14 to 3.4.3 installed, and we recommend getting packages
+directly from `Puppet Labs <http://puppetlabs.com>`_.
+
+Puppet modules must adhere to the Puppet 3.6+ guidelines. Uploaded puppet modules must contain a
```

Comment:
s/guidelines/metadata guidelines, which require a metadata.json file to be present in the puppet module/
Link the 'metadata guidelines' section to https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#publishing-modules-on-the-puppet-forge
Delete the sentence starting with "Uploaded puppet...."